rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -0,0 +1,306 @@
+
+    DefaultError:
+      description: Any other possible errors not currently known.
+      headers:
+        'Content-Version':
+          $ref: '#/components/headers/ContentVersion'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ErrorResponse'
+
+  parameters:
+    AcceptVersion:
+      in: header
+      name: Accept-Version
+      description: >
+        The API version used by the client to produce the REST message. The client will accept
+        responses marked with any minor versions after this one. It does imply that all future
+        minor versions of the message are backward compatible with all previous minor versions of
+        the same message.
+      required: true
+      schema:
+        $ref: '#/components/schemas/Version'
+      examples:
+        major.minor:
+          value: '1.2'
+          summary: A released minor version
+        major.minor.patch:
+          value: '1.0.3'
+          summary: A patched version
+        dev:
+          value: '1.2.0-SNAPSHOT'
+          summary: A development version
+
+  headers:
+    ContentVersion:
+      description: >
+        The API version used by the server to produce the REST message. The server will accept
+        messages for any minor versions prior to this one.
+      required: true
+      schema:
+        $ref: '#/components/schemas/Version'
+      examples:
+        major.minor:
+          value: '1.2'
+          summary: A released minor version
+        major.minor.patch:
+          value: '1.0.3'
+          summary: A patched version
+        dev:
+          value: '1.2.0-SNAPSHOT'
+          summary: A development version
+
+  schemas:
+    Version:
+      description: Version number.
+      type: string
+      minLength: 3
+      maxLength: 80
+      pattern: '^([1-9]([0-9]+)?)\.(0|([1-9]([0-9]+)?))(\.(0|([1-9]([0-9]+)?))(-.*)?)?$'
+      example: 1.2, 2.4.3, 0.2.5-SNAPSHOT
+    TransformRequest:
+      description: Transformation request.
+      type: object
+      required:
+        - id
+        - stagedLocation
+        - productLocation
+        - mimeType
+        - bytes
+        - callbackUrl
+      properties:
+        id:
+          type: string
+          description: The unique identifier for a request.
+          minLength: 1
+          maxLength: 255
+          example: 30f14c6c1fc85cba12bfd093aa8f90e3
+        stagedLocation:
+          type: string
+          description: The temporary location of the ingested product.
+          minLength: 1
+          maxLength: 2048
+          example: https://bucket.s3.amazonaws.com
+        productLocation:
+          type: string
+          description: The final location of the ingested product.
+          minLength: 1
+          maxLength: 2048
+          example: https://bucket.s3.amazonaws.com
+        mimeType:
+          type: string
+          description: The mime type of the ingest product.
```

Comment:
@figliold - ✏️ Should this say "...the *ingested* product" to be consistent with the other descriptions?